include/pm/device.h

```diff
@@ -483,6 +483,36 @@ bool pm_device_state_is_locked(const struct device *dev);
  */
 bool pm_device_on_power_domain(const struct device *dev);
 
+/**
+ * @brief Add a device to a power domain.
+ *
+ * This function adds a device to a given power domain.
+ *
+ * @param dev Device to be added to the power domain.
+ * @param domain Power domain.
+ *
+ * @retval 0 If successful.
+ * @retval -EALREADY If device is already part of the power domain.
+ * @retval -ENOSYS If the application was built without power domain support.
+ * @retval -ENOSPC If there is no space available in the power domain to add the device.
+ */
+int pm_device_power_domain_add(const struct device *dev,
```

Comment:
`int pm_dpd_add(const struct device *dev, const struct device *domain);`,
IMO the abbreviation `_dpd_` for `_device_power_domain_` is obvious.

Reply:
hm it looks to mnemonic for me and does not keep the namespace, maybe `pm_device_pd_add` ?